rasa/core/channels/botframework.py

```diff
@@ -215,6 +215,10 @@ async def health(request: Request) -> HTTPResponse:
 
         @botframework_webhook.route("/webhook", methods=["POST"])
         async def webhook(request: Request) -> HTTPResponse:
+            """ The content of the Activity object exchanged in this request is detailed in MS
+                docs here : https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#activity-object
+
+            """
```

Comment:
Lint is failing here + Added first sentence to describe the purpose of the function.

```suggestion
            """Webhook receiving requests from MS Botframework.

            The request contains data about end-user messages or actions.
            The content of the Activity object exchanged in this request
            is detailed in MS:
            https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#activity-object
```